packages/ui/cypress/tests/network-switch.cy.ts

```diff
@@ -0,0 +1,40 @@
+import { accountDisplay } from '../support/page-objects/components/accountDisplay'
+import { landingPageUrl } from '../fixtures/landingData'
+import { landingPage } from '../support/page-objects/landingPage'
+import { settingsPage } from '../support/page-objects/settingsPage'
+import { testAccounts } from '../fixtures/testAccounts'
+import { topMenuItems } from '../support/page-objects/topMenuItems'
+
+const { name: testAccountName, address: testAccountAddress } =
+  testAccounts['Multisig Member Account 1']
+
+describe('Network can be switched', () => {
+  it('should switch account using menu', () => {
+    cy.visit(landingPageUrl)
+
+    landingPage.watchAccountButton().click()
+
+    cy.url().should('contain', 'settings?network=rococo')
```

Comment:
I'm not sure how pedantic we should be.
having `settings?address=xxx&network=rococo` would fail but be valid. Should we just check for `network=rococo`?

```suggestion
    cy.url().should('contain', 'network=rococo')
```